Update tutorial-define-index.asciidoc (#76975)
elastic/kibana: docs/getting-started/tutorial-define-index.asciidoc

. From the *Time field* dropdown, select *@timestamp, then click *Create index pattern*.
+
[role="screenshot"]

> image::images/tutorial_index_patterns.png[image demonstrating "Create index pattern" button]

Reviewer: ```suggestion
image::images/tutorial_index_patterns.png[Image showing how to create an index pattern]
```
